### ISSUE TITLE
Updating the S3 Paths for the Jupyter Plugin

### ISSUE THREAD

**Issue by marc-rudkowski-tm** [aws-samples/sagemaker-run-notebook#27]
IHAC who is using the Jupyter Plugin in Sagemaker Studio. It is a great plugin, but in the documentation there is a gap for us that I wasn't able to easily figure out in the source code:

Is there a way to change the input/output buckets using the Jupyter Plugin? It seems to use a default Sagemaker bucket. Instead I'd like to scope it to a specific bucket and prefix. Let me know if this is possible.

Cheers!

**Comment by tomfaulhaber** [comment]
Yeah, you can change the code in `upload_fileobj()`([here](https://github.com/aws-samples/sagemaker-run-notebook/blob/master/sagemaker_run_notebook/run_notebook.py#L99)) to specify whatever bucket and prefix you would like.
